jme3-ios/src/main/java/com/jme3/system/ios/JmeIosSystem.java

```diff
@@ -54,6 +55,13 @@
  */
 public class JmeIosSystem extends JmeSystemDelegate {
 
+    public JmeIosSystem() {
+        setErrorMessageHandler((message) -> {
+            showDialog(message);
+            System.err.println("JME APPLICATION ERROR:" + message);
+        });       
+    }
+
```

Comment:
Shouldn't there be a settings handler also provided here (that does nothing and only returns true like the old behavior)?